index.js

```diff
@@ -209,7 +209,10 @@ function encodeTorrentFile (parsed) {
   torrent['url-list'] = parsed.urlList || []
 
   if (parsed.private === true || parsed.private === false) {
-    torrent['private'] = parsed.private
+    // Cast boolean to integer 1 or 0; BitTorrent's bencoding can't encode
+    // booleans.
+    var privateAsInteger = parsed.private & 1
+    torrent['private'] = privateAsInteger
```

Comment:
I prefer to cast the boolean with `torrent['private'] = Number(parsed.private)` which I think is clearer.